frontend/src/containers/DocList.js

```diff
@@ -65,36 +85,32 @@ class DocList extends Component {
   render() {
     const {
       windowType,
-      breadcrumb,
       query,
       modal,
       rawModal,
-      pluginModal,
       overlay,
-      indicator,
       processStatus,
       includedView,
     } = this.props;
-    let refRowIds = [];
+    let refRowIds = EMPTY_ARRAY;
+    const queryCopy = query ? query : EMPTY_OBJECT;
 
-    if (query && query.refRowIds) {
+    if (queryCopy.refRowIds) {
       try {
-        refRowIds = JSON.parse(query.refRowIds);
+        refRowIds = JSON.parse(queryCopy.refRowIds);
       } catch (e) {
-        refRowIds = [];
+        refRowIds = null;
```

Comment:
btw, what's the javascript approach in this case. Don't we need to log the exception or something?

Reply:
Is there a BE 'collector' for such cases ? I recall we talked about adding an API endpoint that would collect FE events also from prod to LogStash

Reply:
no, we don't so far :(

Reply:
Why log an exception ? I've checked how we handle it down the road, and the api request function has `null` as a default for this param. So this shouldn't change anything, and at least we're not passing an empty array all the way.